_apps/commitlint-bot.md

```diff
@@ -0,0 +1,61 @@
+---
+title: commitlint-bot
+description: A GitHub App that lints all commits of new or edited pull requests and
+  sets an appropriate status check.
+slug: commitlint-bot
+screenshots:
+- https://github.com/ahmed-taj/commitlint-bot/blob/master/docs/status.png
+authors:
+- ahmed-taj
+repository: ahmed-taj/commitlint-bot
+host: https://commitlint.now.sh
+stars: 10
+updated: 2017-11-02 17:52:44 UTC
+installations: 1
+organizations:
+- ahmed-taj
+---
+
+<p align="center">
+  <img src="https://github.com/ahmed-taj/commitlint-bot/blob/master/docs/robot.svg" width="256" alt="commitlint-bot logo" />
```

Comment:
Unfortunately hot-linking to SVG images on GitHub doesn't work. It works in READMEs because GitHub will rewrite the links to point to the actual asset, but there's not a way to do it on external sites. You can use the `raw` URL for other image types, but GitHub doesn't serve the proper `Content-Type` headers for SVGs. 😢 

```
$ curl -LI https://github.com/ahmed-taj/commitlint-bot/raw/master/docs/robot.svg
```

Feel free to add the image assets to `/assets/apps/[your-app=name]/` in this repository.

Reply:
I think I will go with GitHub assets for now https://avatars0.githubusercontent.com/in/6357